pkg/db/migrations/202311151856_add_consumers.go

```diff
@@ -22,7 +22,7 @@ func addConsumers() *gormigrate.Migration {
 			}
 
 			if err := CreateFK(tx, fkMigration{
-				"resources", "consumers", "consumer_name", "consumers(name)",
+				"resources", "consumers", "consumer_name", "consumers(name)", "ON DELETE RESTRICT ON UPDATE RESTRICT",
```

Comment:
from SRE comments, we can't modify existing migrations, we need to create new migration to ensure their data can migrated to new schema.